site2/docs/sql-rest-api.md

```diff
@@ -0,0 +1,196 @@
+      "elapsedTimeMillis" : 2,
+      "peakMemoryBytes" : 0
+   }
+}
+
+➜  ~ curl http://localhost:8081/v1/statement/20191113_033653_00006_dg6hb/2 | json_pp
+{
+   "id" : "20191113_033653_00006_dg6hb",
+   "data" : [
+      [
+         "pulsar"
+      ],
+      [
+         "system"
+      ]
+   ],
+   "infoUri" : "http://localhost:8081/ui/query.html?20191113_033653_00006_dg6hb",
+   "columns" : [
+      {
+         "typeSignature" : {
+            "rawType" : "varchar",
+            "arguments" : [
+               {
+                  "kind" : "LONG_LITERAL",
+                  "value" : 6
+               }
+            ],
+            "literalArguments" : [],
+            "typeArguments" : []
+         },
+         "name" : "Catalog",
+         "type" : "varchar(6)"
+      }
+   ],
+   "stats" : {
+      "wallTimeMillis" : 104,
+      "scheduled" : true,
+      "userTimeMillis" : 14,
+      "progressPercentage" : 100,
+      "totalSplits" : 19,
+      "nodes" : 1,
+      "cpuTimeMillis" : 16,
+      "queued" : false,
+      "queuedTimeMillis" : 1,
+      "state" : "FINISHED",
+      "peakMemoryBytes" : 0,
+      "elapsedTimeMillis" : 111,
+      "processedBytes" : 0,
+      "processedRows" : 0,
+      "queuedSplits" : 0,
+      "rootStage" : {
+         "cpuTimeMillis" : 1,
+         "runningSplits" : 0,
+         "state" : "FINISHED",
+         "completedSplits" : 1,
+         "subStages" : [
+            {
+               "cpuTimeMillis" : 14,
+               "runningSplits" : 0,
+               "state" : "FINISHED",
+               "completedSplits" : 17,
+               "subStages" : [
+                  {
+                     "wallTimeMillis" : 7,
+                     "subStages" : [],
+                     "stageId" : "2",
+                     "done" : true,
+                     "nodes" : 1,
+                     "totalSplits" : 1,
+                     "processedBytes" : 22,
+                     "processedRows" : 2,
+                     "queuedSplits" : 0,
+                     "userTimeMillis" : 1,
+                     "cpuTimeMillis" : 1,
+                     "runningSplits" : 0,
+                     "state" : "FINISHED",
+                     "completedSplits" : 1
+                  }
+               ],
+               "wallTimeMillis" : 92,
+               "nodes" : 1,
+               "done" : true,
+               "stageId" : "1",
+               "userTimeMillis" : 12,
+               "processedRows" : 2,
+               "processedBytes" : 51,
+               "queuedSplits" : 0,
+               "totalSplits" : 17
+            }
+         ],
+         "wallTimeMillis" : 5,
+         "done" : true,
+         "nodes" : 1,
+         "stageId" : "0",
+         "userTimeMillis" : 1,
+         "processedRows" : 2,
+         "processedBytes" : 22,
+         "totalSplits" : 1,
+         "queuedSplits" : 0
+      },
+      "runningSplits" : 0,
+      "completedSplits" : 19
+   }
+}
+```
+
+
+
+## Response JSON field
```

Comment:
In the example, there are many fields. why do you only document these two fields?

Reply:
I document these two fields because it's not in the example. And I think it should be known.

I remove the `status` that because of the `status` that is all response data.